### ISSUE TITLE
God of War 3 & Ascension - Sky broken color

### ISSUE THREAD

**Issue by Jonathan44062** [RPCS3/rpcs3#14709]
### Quick summary

When you get to a certain part of the game, you will see how the color of the sky looks weird, in GOW 3 it happens when you go after Hermes, in GOW Ascension, when you go to the Tower of Delphi.

### Details

God of War 3
PS3
![image](https://github.com/RPCS3/rpcs3/assets/78843189/ece6f28c-f30e-4c49-9d1a-de2ff89e2252)

RPCS3
![image](https://github.com/RPCS3/rpcs3/assets/78843189/d64c60c9-8762-4a48-b6ac-76394508604f)

God of War Ascension
PS3
![image](https://github.com/RPCS3/rpcs3/assets/78843189/4d7f2946-4e32-4b61-8d4b-4d76d1640653)

RPCS3
![image](https://github.com/RPCS3/rpcs3/assets/78843189/7c9860a1-5540-4c61-8b7a-fe3b134583f5)

### Attach a log file

GOW 3 Log
[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/12811521/RPCS3.log.gz)

GOW Ascension Log
[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/12811653/RPCS3.log.gz)

### Attach capture files for visual issues

God of War 3 RSX Capture
https://www.mediafire.com/file/idilph8l3bsyhae/BCUS98111_20231004200410_capture.rrc/file

God of War Ascension RSX Capture
https://www.mediafire.com/file/oobxsf01njvwvmw/BCUS98232_20231004202932_capture.rrc/file

### System configuration

Windows 10 22H2
Intel Core i7-8700
15.93 GiB RAM
GPU: NVIDIA GeForce GTX 1070 Ti (537.42)

### Other details

I can share my save files of GOW 3 and GOW Ascension to test better these areas
https://www.mediafire.com/file/10cpopn7eapywbt/GOW_3_Save_file.rar/file
https://www.mediafire.com/file/r6pis3nxm1z2cqm/GOW_Ascension_Save_file.rar/file

**Comment by Asinin3** [comment]
Is this a regression

**Comment by Jonathan44062** [comment]
No, it was always like this, but no one noticed the problem

**Comment by kd-11** [comment]
Isn't this a known issue with one of the GOW3 patches?

**Comment by Jonathan44062** [comment]
No, the screenshot I posted is without the patches (If you use the patch Disable Layered Fog, the broken sky is removed) 
Record a little to appreciate the issue
https://drive.google.com/file/d/1YLETrZ3_yyKmVynp-RQr-wJ6wudDLF-w/view?usp=sharing

**Comment by kd-11** [comment]
> No, the screenshot I posted is without the patches (If you use the patch Disable Layered Fog, the broken sky is removed) Record a little to appreciate the issue https://drive.google.com/file/d/1YLETrZ3_yyKmVynp-RQr-wJ6wudDLF-w/view?usp=sharing

Thanks. I tried loading the save, but I have the US disc version for GOW3. I'll try and source the relevant one since these games have complex rendering that breaks RSX captures.

**Comment by digant73** [comment]
still broken in current rpcs3 release

**Comment by amiMohammad** [comment]
Same for me :(                      RPCS3 Build 0.0.37-18022

**Comment by JackVerse8** [comment]
Is this bug fixable?

**Comment by digant73** [comment]
@Jonathan44062 you recently closed the issue on one cutscene (missing texture on Kratos) as due to MLAA patch. Did you test if even this issue was affected by that patch? In my case, the game never starts without the patch so I cannot test by myself

**Comment by Jonathan44062** [comment]
Same as always, no patches
<img width="2559" height="1391" alt="Image" src="https://github.com/user-attachments/assets/0cc70d53-a7d4-451a-884d-0419cfee4fd4" />

<img width="376" height="293" alt="Image" src="https://github.com/user-attachments/assets/3ced238d-0d31-4830-a07a-21442a9f32c1" />

This also applies for GOW3

**Comment by Colotov-Mocktail** [comment]
Still broken in current release.

<img width="1920" height="1040" alt="Image" src="https://github.com/user-attachments/assets/c10475f1-b358-44ea-b86b-a642e84c316e" />

**Comment by treeleaves12** [comment]
Still happening, I have BCES00510 v1.03 and using rpcs3 19185. It is like the smoke or fog floating moving in the air that gets the wrong color. Weird bug as you have to get halfway in the game to make it appear. 
Using default settings except framelimit 60 and wcb enabled. Win11, intel i5-13600K, geforce 3070. I tried to also enable the following settings (SPU Xfloat Accuracy: Accurate, Strict Rendering Mode, Read Color Buffers, Write Depth Buffer, Read Depth Buffer, Handle RSX Memory Tiling, Accurate SPU DMA, Accurate RSX Reservation Access), but unfortunately they didn't fix this.

Not sure if useful but just in case I also took the log and rsx capture:
https://mega.nz/file/G9BBBDiB#3vYnBbkiNA6lABrHT6JHvnj5Q6asm8_NFWF5au9Wz1U

**Comment by Xcedf** [comment]
That's the problem with the Layered Fog, that's how it called inside executable
Enabled
<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/aa7fdb31-30d8-417d-8869-6f983ad23159" />
Disabled
<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/6c5560d4-3350-4ddb-ad32-6ce7766816d8" />
So I guess wrong colors for some layer of the depth, same issue for GoW3 as well

**Comment by albertmleczak** [comment]
Will this patch be available in the future like in GoW3?

**Comment by Longmao0219** [comment]
Won't change with RPCS3 0.0.40-19243-a60734e2, use patch Disable Layered Fog can slove it but some distant scenes are too clear, it disrupted the original atmosphere of the scene.

**Comment by albertmleczak** [comment]
I meant to ask if there will also be a "Disable Layered Fog" patch in "God of War Ascension" like there is in "God of War 3".
